mono/utils/checked-build.h

```diff
@@ -12,6 +12,10 @@
 
 #include <config.h>
 
+// This is for metadata writes which do not have to be checked.
+// We still use the macro so that they are easier to find later.
```

Comment:
An explanation on what the circumstances are under which a write should be exempt would be nice.